components/brave_wallet/browser/zcash/rust/BUILD.gn

```diff
@@ -46,7 +46,10 @@ source_set("orchard_impl") {
 }
 
 rust_static_library("rust_lib") {
-  visibility = [ ":orchard_impl" ]
+  visibility = [
+    ":orchard_impl",
+    "//brave/components/brave_wallet/browser",
```

Comment:
the entire point of `orchard_impl`  is to encapsulate the rust cxx types so they don't get exposed to other cpp code and that's why this has visibility so nothing else should directly depend on this

Reply:
Done